source/gameengine/GamePlayer/GPG_ghost.cpp

```diff
@@ -1090,11 +1155,25 @@ int main(
 							bfd = load_game_data(temppath);
 						}
 					}
-					else {
-						bfd = load_game_data(BKE_appdir_program_path(), filename[0]? filename: nullptr);
-						// The file is valid and it's the original file name.
-						if (bfd) {
-							KX_SetOrigPath(bfd->main->name);
+					else
+					{
+#ifdef WITH_GAMEENGINE_BPPLAYER
+						if (useLocalPath) {
+							bfd = load_encrypted_game_data(filename[0] ? filename : NULL, localFilePath, hexKey);
+
+							// The file is valid and it's the original file name.
+							if (bfd) {
+								remove(filename);
```

Comment:
Because that file contains the encryption keys. Once saved them it is better to remove it to mitigate possible key capture/cheats